open-metadata-resources/open-metadata-archives/design-model-archives/src/main/java/org/odpi/openmetadata/archiveutilities/designmodels/test/JenaTest.java

```diff
@@ -0,0 +1,549 @@
+/* SPDX-License-Identifier: Apache 2.0 */
```

Comment:
Seems this is in the wrong package - should be in the OWL package

Reply:
I think you are looking at an old version of the fix this file is not in the current fix